integration/integration.go

```diff
@@ -0,0 +1,36 @@
+package integration
+
+import (
+	"github.com/lacework/go-sdk/api"
+	"log"
+	"os"
+	"strings"
+)
+
+var LwIntegrationClient *api.Client
```

Comment:
nit: maybe better name?
```suggestion
var LwClient *api.Client
```